config/Config-build.in

```diff
@@ -328,4 +328,14 @@ menu "Global build settings"
 			bool "Full"
 	endchoice
 
+	config CONFIG_TARGET_ROOTFS_SECURITY_LABELS
+		bool "Enable rootfs security labels"
+		select KERNEL_SQUASHFS_XATTR
+		select KERNEL_EXT4_FS_SECURITY
+		select KERNEL_F2FS_FS_SECURITY
+		select KERNEL_UBIFS_FS_SECURITY
+		select KERNEL_JFFS2_FS_SECURITY
+		select PACKAGE_refpolicy
+		help
+		  This option allows the usage of SELinux labels
```

Comment:
`s/allows/enables` (especially as this effectively turns on labels in squashfs and installs refpolicy, that's more than 'allowing').

Reply:
Fixed with latest push.